app/Server.hs

```diff
@@ -62,6 +62,7 @@ runServer = do
               dir "get-json-data" $ look "graphName" >>= \graphName -> liftIO $ getGraphJSON graphName,
               dir "loading" $ look "size" >>= loadingResponse,
               dir "save-json" $ look "jsonData" >>= \jsonStr -> look "nameData" >>= \nameStr -> liftIO $ saveGraphJSON jsonStr nameStr,
+              dir "export-pdf" $ exportResponse "HelloWorld",
```

Comment:
You can start by making this use the same image from the `image` route.

In order to do the `timetable-image` route, you'll need to add `look` calls to access the "courses" and "session" query parameters.